common/changes/@uifabric/experiments/yudon-tooltipbug0315_2019-03-15-23-05.json

```
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@uifabric/experiments",
+      "comment": "fix a bug in the tooltip of last page button in pagination component, experiment package",
+      "type": "minor"
```

```suggestion
      "type": "patch"
```

This change should be of type `patch` as you are not modifying any APIs, simply patching a bug.

---

Updated.